### PR TITLE
Add Keystore Datamatrix Print Overlay (Export)

### LINKED ISSUE #2202: Datamatrix Export View

### PR DESCRIPTION
**Pull request by Akamig**
Resolves #2202

(Still working on UI)

### CODE REVIEW

File: mead.config.json
Comment (longfin) [inline mead.config.json:1]: This file doesn't seem to be needed for #2202

File: bitgener.d.ts
Comment (longfin) [inline bitgener.d.ts:1]: it might be better than publishing this type as a separate package.

Or contributing upstream can be an option.

Comment (Akamig) [reply bitgener.d.ts:1]: I talked to maintainer and It seems they want to keep their library totally pure javascript. so....

File: mead.config.json
Comment (Akamig) [reply mead.config.json:1]: Oh, That was my fault.

File: bitgener.d.ts
Comment (longfin) [reply bitgener.d.ts:1]: Then, how about publishing on [DT](https://definitelytyped.github.io/)?

Comment (Akamig) [reply bitgener.d.ts:1]: Alright!

**Review by 24pAs — APPROVED** [review]

**Comment by Akamig** [comment]
We should hold this PR until concrete representation approved by publishing team.

**Pull request merged**
